data.php

```diff
@@ -0,0 +1,21 @@
+<?php
+require_once 'function.php';
+function addFormToDataBase() {
+    if(isset($_POST['name']) && isset($_POST['img']) && isset($_POST['brand']) && isset($_POST['price'])) {
+        $nameInput = $_POST['name'];
+        $imgInput = $_POST['img'];
+        $brandInput = $_POST['brand'];
+        $limitedInput = isset($_POST['limited']) ? 1 : 0;
+        $priceInput = $_POST['price'];
```

Comment:
Definitely want to add some basic validation here, for example making sure that the name input isn't empty (I managed to add a nameless deck of cards by just typing spaces in.

I'd run all of your string based inputs through `trim()` and `htmlspecialchars()` before making sure they actually contain characters.

One thing you could do is add error messages into the header redirect which you could then display on the index.php file.

Something like `header('Location: index.php?error=Name must contain more than 3 characters');`